Unix/base/batch.h

```diff
@@ -113,68 +113,173 @@ void Batch_InitFromBuffer(
 void Batch_Destroy(
     Batch* self);
 
-Batch* Batch_New(
-    size_t maxPages);
-
 void Batch_Delete(
     Batch* self);
 
-void* Batch_Get(
+#if defined(CONFIG_ENABLE_DEBUG) && defined(CONFIG_OS_LINUX)
+
+#define BATCH_CALLSITE __FILE__, __LINE__, __FUNCTION__
```

Comment:
if you defined this with a trailing comma, you would not need to duplicate the Batch_* functions from debug and not

Reply:
Fixed.